src/schema.graphql

```diff
@@ -43,6 +44,10 @@ type Mutation {
   createOrganization(nameEn: String!, nameFr: String!, acronymEn: String!, acronymFr: String!): Organization!
   modifyOrganization(id: ID!, data: ModifyOrganizationInput): Organization!
   deleteOrganization(id: ID!): Boolean!
+  createApproval(gcIDApprover: gcIDProfileInput!, gcIDSubmitter: gcIDProfileInput!, requestedChange: RequestedChangeInput, createdOn: String, actionedOn: String, deniedComment: String, status: Status, changeType: ChangeType): Approval!
```

Comment:
Can remove the declaration of the mutation.  The createApproval function will be called internally.  No need to publish it to endpoint.